crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -555,33 +554,30 @@ impl OlmMachine {
         signing_key: &str,
         event: &RoomKeyEvent,
     ) -> OlmResult<Option<InboundGroupSession>> {
+        let unsupported_warning = || {
```

Comment:
What's the point of this closure?

Reply:
There's little point here currently, but this whole PR was picked up from a bigger set of changes and we'll eventually have multiple branches that will use this closure.

Reply:
Okay.